Content.Shared/Sound/SoundSpecifierTypeSerializer.cs

```diff
@@ -1,17 +1,29 @@
 using System;
+using Content.Shared.Audio;
+using Robust.Shared.ContentPack;
 using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
 using Robust.Shared.Serialization.Manager;
 using Robust.Shared.Serialization.Manager.Attributes;
 using Robust.Shared.Serialization.Manager.Result;
 using Robust.Shared.Serialization.Markdown.Mapping;
 using Robust.Shared.Serialization.Markdown.Validation;
+using Robust.Shared.Serialization.Markdown.Value;
+using Robust.Shared.Serialization.TypeSerializers.Implementations;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 using Robust.Shared.Serialization.TypeSerializers.Interfaces;
 
 namespace Content.Shared.Sound
 {
     [TypeSerializer]
-    public class SoundSpecifierTypeSerializer : ITypeReader<SoundSpecifier, MappingDataNode>
+    public class SoundSpecifierTypeSerializer :
+        ITypeReader<SoundSpecifier, MappingDataNode>,
+        ITypeReader<SoundSpecifier, ValueDataNode>
     {
+        private readonly ResourcePathSerializer _resourcePathSerializer = new();
+        private readonly PrototypeIdSerializer<SoundCollectionPrototype> _prototypeIdSerializer = new();
```

Comment:
no bad.

just Read<ResourcePath> and Read<SoundCollectionPrototype> in the read, write<...> in write, etc

Reply:
done, you should probably document this somewhere